### PR TITLE
Feature/geekodoc (fixes issue #232)

### PR DESCRIPTION
**Pull request by sknorr**
@tomschr Can you look at this again..?

@fsundermeyer Can you look at  fdc375e in particular and make sure I did not make too much of a mess of the Makefile? I did not touch the `dist` and `install` targets. However, `dist` now fails with an error mentioning an `.ent` file...
Normal make runs through, seemingly without issues.

(Update toms: related to #232)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,17 +25,17 @@ SUSE_STYLES_PATH := $(DB_XML_PATH)/stylesheet
...
+NOVDOC_DTD_PATH := $(SUSE_SCHEMA_PATH)/NovDoc/dtd/$(NOVDOC_VERSION)
 
 #--------------------------------------------------------------
-# SUSEDOC
+# GEEKODOC
 
-#SUSEDOC_NAME     := susedoc
-#SUSEDOC_VERSION  := 0.9
-#SUSEDOC_RNG_PATH := $(SUSE_SCHEMA_PATH)/rng/$(SUSEDOC_VERSION)
+#GEEKODOC_NAME     := geekodoc
```
Comment (tomschr) [inline Makefile]: If you want to use these names in the future, maybe use a more general name like `SUSESCHEMA`? Something I did in my Makefile for GeekoDoc. ;-)

**Comment by tomschr** [comment]
@sknorr It seems we miss a `catalog.xml` file in `schema/GeekoDoc/rng/`. Additionally we need to agree on an identifier. Any ideas?

A (maybe?) minor issue: do we really want to use capital letters in our directories? Do we want to use `schema/geekodoc` and `schema/novdoc` instead of the current version? Just asking. IMHO, the lowercase version is a bit easier to type. ;-)

**Comment by tomschr** [comment]
> However, dist now fails with an error mentioning an .ent file...

I think I found the problem: it was a link pointing to the wrong target. Fixed in 334a600

**Comment by sknorr** [comment]
@tomschr Can you have a look again and merge if it's fine?

**Pull request closed without merging**